src/jit/lower.cpp

```diff
@@ -23,6 +23,7 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
 #ifndef LEGACY_BACKEND // This file is ONLY used for the RyuJIT backend that uses the linear scan register allocator
 
+#include "sideeffects.h"
 #include "lower.h"
```

Comment:
IMO, you should have this `#include` in lower.h, instead of spreading it around to all the cpp files that happen to include lower.h